### DISCUSSION TITLE
database realtime subscriptioin NO NEW DATA IN CONSOLE LOG

### DISCUSSION THREAD

**Discussion by bobwatcherx** [supabase/supabase#5439] [Questions] [answered]
i have problem with realtime subscription 
. i click button to add new data ,  but i get new console log like here
, but no **NEW DATA INPUT VALUE FROM TEXTINPUT** in Console log

my code :

`
async function handlepost(){
	const { data, error } = await supabase
  .from('tbltest')
  .insert([
    { nama:nama }
  ])
  **getrealtime()**

}
function getrealtime(){
	const mySubscription = supabase
  .from('tbltest')
  .on('*', payload => {
    console.log('Change received!', payload.) **<----- NOT SHOW IN MY CONSOLE LOG** 
  })
  .subscribe()

  const subscriptions = supabase.getSubscriptions()
  console.log("subs",mySubscription)
  console.log("dann",subscriptions)
}`
my console log
```
RealtimeSubscription {topic: 'realtime:public:tbltest', params: {…}, socket: RealtimeClient, bindings: Array(7), state: 'joining', …}
bindings: (6) [{…}, {…}, {…}, {…}, {…}, {…}]
joinPush: Push {channel: RealtimeSubscription, event: 'phx_join', payload: {…}, timeout: 10000, sent: true, …}
joinedOnce: true
params: {user_token: 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJyb2xlIjoiY…DQ4fQ.9A3uBTWE5yztfnyrVdvQb1WM_IuvIimbmM3SU3PcNZM'}
pushBuffer: []
rejoinTimer: Timer {timer: undefined, tries: 0, callback: ƒ, timerCalc: ƒ}
socket: RealtimeClient {accessToken: null, channels: Array(0), endPoint: 'wss://kjdfchehyjheuutobjdb.supabase.co/realtime/v1/websocket', headers: {…}, params: {…}, …}
state: "closed"
timeout: 10000
topic: "realtime:public:tbltest"
[[Prototype]]: Object
```

**Comment by GaryAustin1** [comment, marked as answer]
At least one problem looks to be you call getrealtime() function after you have done the insert.  Your realtime connection has to be running before any data changes.  If you change the table in the UI do you get a subscription change?  Also make sure RLS if on allows a select for the user and replication is turned on in the database section of the UI for that table.
